genesis/options/vis.py

```diff
@@ -127,3 +130,7 @@ def __init__(self, **data):
             gs.raise_exception(
                 f"Unsupported `render_particle_as`: {self.render_particle_as}, must be one of ['sphere', 'tet']"
             )
+
+        if not self.n_rendered_envs is None:
+            assert self.rendered_envs_idx is None
```

Comment:
Add a deprecation warning. This has been done recently for the contact resolve time:
https://github.com/Genesis-Embodied-AI/Genesis/blob/53b475f49c025906a359bc8aff1270a3c8a1d4a8/genesis/engine/solvers/rigid/rigid_solver_decomp.py#L75